jac-uk/apply
# 1253 Include qualification certificates in candidate profile

## What's included?
Store the practising/exemption certificates alongside the rest of the qualifications data so that the candidate doesnt have to upload them each time they make a new application.

Closes #1253 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the following url and login: https://jac-apply-develop--pr1276-feature-1253-include-sdcqc0mu.web.app/
- Apply for a new vacancy
- Go to the relevant qualifications and specify that you are a barrister who has NOT completed pupillage. You will be given the opportunity to upload exemption and practicing certificates. Upload an exemption one only and press 'Save and Continue'
- Apply for a different vacancy
- Go to the relevant qualifications and you should see the form prepoulated and the exemption certificate should be there

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
